code/style.css

```diff
@@ -1,19 +1,60 @@
 body {
   margin: 0;
   font-family: "Courier New", Courier, monospace;
+  background-image: url(./assets/image-main-dark5.jpg);
```

Comment:
It's far more safe approach to wrap path in parentheses. Without them, some browsers might not find the path
`url("./assets/image-main-dark5.jpg");`